### ISSUE TITLE
Observe events when 2 Chrome plugins use Gmail.js cause Gmail to enter error mode

### ISSUE THREAD

**Issue by derekribbons** [KartikTalwar/gmail.js#265]
Hello,

I've noticed that when the following code is executed by 2 extensions using gmail.js, Gmail will enter its "retry" error state after a couple clicks.

gmail.observe.on('open_email', function() {
    console.log("email opened"); 
}); 

Other functions work correctly, it seems to be tied to the observer code.  Have you seen this behavior before?  Any suggestions on ways to fix / work around it?

I've tried namespacing the my instance of gmail.js, monkeying with the event names, still get the dreaded "something broke" Gmail dialog and retries.

Thanks,
-Derek

**Comment by JerryBels** [comment]
Hum... It could be linked to the error I opened here : https://github.com/KartikTalwar/gmail.js/issues/238
which makes me think, maybe the error I reported there is not the root of the issue. Combining with your reported issue, it may be simply that observers in general are trigerring errors in Gmail when there is 2 of them on the same event !

**Comment by derekribbons** [comment]
Thanks for the reply Jerry,

We had a similar issue in our plugin a while back, so I broke our init up to a once and an always section.  Since we inject a full page into gmail based on different events, the observers would fail, I moved them into the once and everything else into the always piece of our init and things were happy.  Then another group deployed a plugin using Gmail.js (we don't have a lot of control over that plugin) and we were back to square one.  Happy to help or try things, let me know.
